### ISSUE TITLE
Build this project (and its build project) with our custom 2.10 flags.

### ISSUE THREAD

**Issue by sellout** [slamdata/sbt-slamdata#14]
Perhaps use `commonBuildSettings` in project/build.sbt?

**Comment by djspiewak** [comment]
@sellout Can you double-check this?  I think @drostron did this last week and we forgot to change this issue.

**Comment by sellout** [comment]
@djspiewak I don’t think this has changed. If it were, I’d expect `slamdata.SbtSlamData.commonBuildSettings` to be referenced somewhere in project/build.sbt.

But I could be wrong about that …

**Comment by drostron** [comment]
We're utilizing `commonBuildSettings` in downstream projects as well as Quasar's build project. 

They are not yet applied to sbt-slamdata itself.

**Comment by djspiewak** [comment]
@drostron Something I was thinking about: I think we should split off the minimal subset of the compiler plugin which we want to apply to meta-projects, and then apply that minimal subset (as well as include) to the broader plugin.  This would avoid situations like in quasar where we're including `sbt-slamdata` en bulk in both the project and the meta-project, where we really only want the compiler options for the meta-project.

**Comment by sellout** [comment]
@djspiewak But _do_ we only want to apply the compiler settings? I mean, currently that’s all we are applying, but why not use everything on the metaproject? Especially something like WartRemover.

**Comment by djspiewak** [comment]
@sellout But do we want *everything*?

**Comment by drostron** [comment]
@djspiewak perhaps. My understanding of direction, as was in place in quasar prior to sbt-slamdata, is that a meta-project build was just another build where we want to apply our usual build settings. `commonBuildSettings` already has the beginnings of handling various scala version flavors and could be refined further.

**Comment by sellout** [comment]
@djspiewak Well, as @drostron said, the idea of this is to set up all our defaults – basically, what we think the default behavior of scalac, sbt, etc. should be anyway. So my inclination is that yes, we do want everything, unless there is some particular problem that arises.
